src/opensearchDatasource.ts

```diff
@@ -647,6 +651,18 @@ export class OpenSearchDatasource
   }
 
   query(request: DataQueryRequest<OpenSearchQuery>): Observable<DataQueryResponse> {
+    console.log('[OpenSearchDatasource] query', request);
+
+    if (request.liveStreaming && !this.isHandlingLiveStreaming) {
```

Comment:
Wondering, when is `this.isHandlingLiveStreaming` relevant for the if statement? I added a breakpoint to line 656 and ran the query and I don't see it being anything other than equal to request.liveStreaming at any point `query` is invoked, but I might be missing something